app/src/ui/changes/filter-changes-list.tsx

```diff
@@ -1454,37 +1533,53 @@ export class FilterChangesList extends React.Component<
   }
 
   private getNoResultsMessage = () => {
-    if (
-      this.props.filterText === '' &&
-      !this.props.includedChangesInCommitFilter
-    ) {
-      return undefined
-    }
-
-    const filterTextMessage = this.props.filterText
-      ? ` matching your filter of '${this.props.filterText}'`
-      : ''
-
-    const includedCommitText = this.props.includedChangesInCommitFilter
-      ? ' that are to be included in your commit'
-      : ''
-
-    const conjunction = filterTextMessage && includedCommitText ? ' and ' : ''
-
-    return `Sorry, I can't find any changed files${filterTextMessage}${conjunction}
-        ${includedCommitText}.`
+    return getNoResultsMessage(this.props.filterText, {
```

Comment:
Nit: Once the filter state object is passed down as a prop, I think it won't make sense to separate out the `filterText` and simply pass in a filters object.